extensions/amp-autocomplete/0.1/test/validator-amp-autocomplete.html

```diff
@@ -18,11 +18,11 @@
   Tests for the amp-autocomplete tag.
 -->
 
-<!doctype html>
+<!DOCTYPE html>
 <html ⚡>
-<head>
-  <meta charset="utf-8">
-  <title>amp-autocomplete example</title>
+  <head>
+    <meta charset="utf-8" />
```

Comment:
Where do you see that? https://html.spec.whatwg.org/multipage/parsing.html#self-closing-start-tag-state seems to indicate it's a perfectly valid state (nothing mentioned about parse errors).

Reply:
This styling was updated automatically via `gulp prettify`, happy to remove to restore the document as it was.

Reply:
This is interesting. `meta` is a [void element](https://html.spec.whatwg.org/multipage/syntax.html#void-elements) in the HTML spec, which means that it will technically parse properly even when written as a self-closing tag.

The [Google Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Document_Type) explicitly says that we should "not close void elements".

Prettier on the other hand, has stuck with always closing void elements and there is a lively debate on if they should allow customization of this: https://github.com/prettier/prettier/issues/5246.

My preference is to go against our own style guide and stick with tooling for now (self-closed)

Reply:
@twifkak, at the link you sent, if I'm reading it correctly, reads that the parser should consume the next character. If the token is `>` OK, any other character is considered a parse error. Yes, there are self-closing tags. `<meta>` is one of them, as @samouri noted. However, there is no self-closing tag syntax. A simple test of this is to render a page with `<b/>foo`. Foo will be bold, due to the fact that the `/` is ignored as a parse error.

Reply:
@Gregable Click on the h5 to view its referrers. In particular, https://html.spec.whatwg.org/multipage/parsing.html#after-attribute-value-(quoted)-state:self-closing-start-tag-state. Yes, `<foo/>` gets tokenized as if it were `<foo>`; it's still valid, though. The parser interprets `<foo>` accordingly based on its voidness.